packages/integration-tests/src/reusable-tests/index.ts

```diff
@@ -0,0 +1,2 @@
+export { deployE2eFlow } from './deploy_e2e_flow.js';
```

Comment:
we can just import directly from the file where these functions are defined, no need for an index file